website/docs/tutorial/1b-setting-up-snowflake.md

```diff
@@ -309,4 +314,11 @@ Awesome!  You are all set to leverage the power of dbt Cloud and Snowflake toget
 * Loaded training data into your Snowflake account
 * Created a dbt Cloud account, either through Partner Connect or through the account flow
 * Connected dbt Cloud and Snowflake
-* Set up the dbt Cloud IDE, queried data, and did your first dbt run
\ No newline at end of file
+* Set up the dbt Cloud IDE, queried data, and did your first dbt run
+
+### Create dbt Cloud account
+
+### Enter connection credentials
+
+## Next steps
```

Comment:
Maybe I'm missing the use of the headers without text underneath?

Reply:
Content was just not ready yet!